doc/man3/OSSL_PROVIDER.pod

```diff
@@ -101,6 +107,11 @@ loaded and initialized or if I<retain_fallbacks> is nonzero.
 If the provider loads successfully and I<retain_fallbacks> is zero, the
 fallback providers are disabled.
 
+OSSL_PROVIDER_load_ex() and OSSL_PROVIDER_try_load_ex() are the variants
+of the previous functions accepting C<OSSL_PARAM> array of the parameters
+that are passed as the configuration of the loaded provider. The parameters
+of any type but C<OSSL_PARAM_UTF8_STRING> are silently ignored.
```

Comment:
This should describe what happens if parameters are supplied in both the config file *and* via these functions.

Reply:
I believe that we don't get there from the config-provided code path though I am not sure

Reply:
I believe that if the a provider is configured in the config file but *without* an "activate" line then the parameters supplied via `OSSL_PROVIDER_load_ex()` are *added* to those from the config file (in the way that you have currently written this).

Reply:
Provided a clarification.